src/components/Desk.tsx

```diff
@@ -0,0 +1,50 @@
+import { useYDoc } from "zustand-yjs";
+import { YJS_ROOM } from "../constants";
+import { LetterSharedData, LetterInterface } from "../types";
+import { Letters } from "../data/letters";
+import { useEffect, useState } from "react";
+import { LetterFormButton } from "./LetterForm";
+import { Letter } from "./Letter";
+import { ShuffleIcon, ResetIcon, ViewGridIcon } from "@radix-ui/react-icons";
+import { connectDoc } from "../utils/yjs";
+import { SubmitLetterMetadata } from "../App";
+
+export function Desk() {
+  const yDoc = useYDoc(YJS_ROOM, connectDoc);
+  const sharedMap = yDoc.getMap<LetterSharedData>("shared");
+  const [letters, setLetters] = useState<LetterInterface[] | undefined>();
+
+  useEffect(() => {
+    setLetters([
+      ...Letters,
+      {
+        ...SubmitLetterMetadata,
+        ctaContent: <LetterFormButton letter={Letters[0]}></LetterFormButton>,
+      },
+    ]);
+  }, []);
+
+  return (
+    <>
+      <div className="toolbar">
+        {/* reset letters */}
+        <button>
```

Comment:
these aren't linked yet, are they?

Reply:
not yet ill comment out so it doesnt show on website for now